common/source/docs/common-ac2_simple_geofence.rst

```diff
@@ -98,32 +137,30 @@ for firmware versions before 4.0:
 .. image:: ../../../images/Fence_MPCh78.png
     :target: ../_images/Fence_MPCh78.png
    
-alternatively, for firmware versions 4.0 or later, an RCx_OPTION can be set via the Config/Tuning > Full Parameter List screen:
+Alternatively, for firmware versions 4.0 or later, an RCx_OPTION can be set via the Config/Tuning > Full Parameter List screen:
 
 -  Use an RCx_OPTION set to Fence
--  holding the switch high (i.e. PWM > 1800) will enable the fence, low
+-  setting the switch high (i.e. PWM > 1800) will enable the fence, low
    (under 1800) will disable the fence.
 
 
 Warnings:
 =========
 
 -  The minimum recommended fence radius is 30m
--  The fence requires the GPS to be functioning well so do not disable
+-  The fence requires the GPS to be functioning well so do not disable.
    the :ref:`GPS arming check <common-prearm-safety-checks>` nor the :ref:`EKF failsafe <ekf-inav-failsafe>` while the fence is enabled. 
```

Comment:
Sentence on line 151 was ended with Full stop - capital letter starting the next sentence. Should this also be a new dot point?

```suggestion
-  The :ref:`GPS arming check <common-prearm-safety-checks>` nor the :ref:`EKF failsafe <ekf-inav-failsafe>` while the fence is enabled. 
```

Reply:
fixed